build/tools/templates/Archive/Archive.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
.Web? Or can this be just 'microsoft.net.sdk'?

Reply:
Maybe, I just copied this over from MetaPackages. I'll change it and see if it works.